resources/views/admin/modal/deletecategory.blade.php

```diff
@@ -0,0 +1,23 @@
+<!-- Modal -->
+<div class="modal fade" id="deleteCategpryModal" tabindex="-1" aria-labelledby="deleteCategpryModalLabel" aria-hidden="true">
```

Comment:
Typo "Categpry" => "Category", might be confusing in the future

Reply:
Sorry, I did not notice the typo. I will correct it and send the pull request again. Thanks for letting me know.